doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -467,6 +468,58 @@ int main(int argc, char** argv)
   /* Wait for MoveGroup to receive and process the attached collision object message */
   visual_tools.prompt("Press 'next' in the RvizVisualToolsGui window once the new object is detached from the robot");
 
+  // Moving to a goal pose with orientation tolerances
+  // ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Getting some "WARNING: Title underline too short" warnings here when building this locally. But the site looks okay.

Reply:
solved